File: Android/Java/app/src/main/java/com/unity3d/levelplaydemo/DemoActivityListener.java
package com.unity3d.levelplaydemo;

import com.unity3d.mediation.rewarded.LevelPlayReward;

public interface DemoActivityListener {
    void setEnablementForButton(DemoButtonIdentifiers buttonIdentifier, boolean enable);
    void setBannerViewVisibility(int visibility);
    void setReward(LevelPlayReward reward);
    void showRewardDialog();
    void createInterstitialAd();
    void createRewardedAd();
    void createBannerAd();
}

File: Android/Kotlin/app/src/main/java/com/unity3d/levelplaydemo/DemoActivityListener.kt
package com.unity3d.levelplaydemo

import com.unity3d.mediation.rewarded.LevelPlayReward

interface DemoActivityListener {
    fun setEnablementForButton(buttonIdentifier: DemoButtonIdentifiers, enable: Boolean)
    fun setBannerViewVisibility(visibility: Int)
    fun setReward(reward: LevelPlayReward)
    fun showRewardDialog()
    fun createInterstitialAd()
    fun createRewardedAd()
    fun createBannerAd()
}